.rubocop.yml

```diff
@@ -1,3 +1,3 @@
 # This is the rubocop configuration file for this gem.
 # Repos using this gem should inherit from .default.yml, not this file.
-inherit_from: .default.yml
+inherit_from: .default.yml
```

Comment:
Please back out all changes to this file. It looks like your editor has removed the final newline, which we try to avoid. Normally, Rubocop would flag the missing newline as an offense, but Rubocop isn't run on `.rubocop.yml`. I recommend updating your editor's settings to ensure newlines are added at the end of files. See:
https://rubystyle.guide/#newline-eof
https://thoughtbot.com/blog/no-newline-at-end-of-file

Reply:
added new line EOF